core/boards/nrf52840_pca10059-sub-slots-stock.dts

```diff
@@ -27,9 +27,14 @@
 			reg = <0x00030000 0x00003E000>;
 		};
 
-		ot_partition: partition@fe000 {
+		/*
+		 * Using 4 pages of scratch partition for OpenThread storage.
```

Comment:
I suggest you specify which pages the scratch is using.

Reply:
Done